### PR TITLE
Update misc tools

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: docker/build-tools/Dockerfile
Comment (ericvn) [inline docker/build-tools/Dockerfile:45]: 2.25.1

**Review by ericvn — DISMISSED** [review]

**Comment by ericvn** [comment]
Hugo was updated in netlify with https://github.com/istio/istio.io/pull/12903

**Comment by ericvn** [comment]
I expect we want to hold off on merging this and updating build-tools until after the istio-private work has completed, likely April 5.

**Comment by ericvn** [comment]
Let's merge this and then when the Go updates are ready they can merge on top.

**Pull request merged**
